### ISSUE TITLE
[SSR:skip@3.0.0-beta.50]: query is still firing with skip set to true (server-side only)

### ISSUE THREAD

**Issue by nathanschwarz** [apollographql/apollo-client#6342]
I have an issue with the new beta release `@3.0.0-beta.50` (server-side only) concerning `useQuery` with `skip: true`.

**Intended outcome:**
the queries are skipped.

**Actual outcome:**
The queries with `skip: true` option are still firing, leading to unwanted errors.

**Comment by joziahg** [comment]
Having same issue with regular react.
```
    "@apollo/react-hooks": "^3.1.5",
    "apollo-cache-inmemory": "^1.6.6",
    "apollo-client": "^2.6.9",
```

**Comment by buzinas** [comment]
See https://github.com/apollographql/react-apollo/issues/3492#issuecomment-644385154. The issue occurs on the client as well.

**Comment by MichaelBurgess** [comment]
Seeing this on `3.0.0` / `3.0.1` on the client side also. Interestingly, if you have no variables, or variables set to `{}`, then the query is skipped as expected.

**Comment by 3nvi** [comment]
This got introduced at a late beta of 3.0 in the client as well.

**Comment by nathanschwarz** [comment]
@hwillson is this patched with #6752 ?
thanks !

**Comment by hwillson** [comment]
Let us know if this is still a concern with `@apollo/client@latest` - thanks!
